helper_scripts/package_windows.py

```diff
@@ -1,15 +1,55 @@
 import os
-
+import sys
+import subprocess
 
 try:
     import PyInstaller
 
-    if PyInstaller:
-        print("PyInstaller is good to go!")
+    print("PyInstaller is good to go!")
 except ModuleNotFoundError:
     raise Exception(
         "You need pyinstaller installed. Make sure to install a version at least 2 months old as to avoid false positives on virus detectors. Use pip install pyinstaller."
     )
 
-os.chdir("src")
-os.system("pyinstaller ../helper_scripts/win.spec")
+# Get the absolute path of the script
+script_dir = os.path.dirname(os.path.abspath(__file__))
+print(f"Script directory: {script_dir}")
+
+# Find the correct paths
+project_root = os.path.dirname(os.path.dirname(script_dir))
+src_path = os.path.join(project_root, "src")
+spec_path = os.path.join(script_dir, "win.spec")
+
+print(f"Project root: {project_root}")
```

Comment:
We probably don't need to print these out. Most likely, these won't change.

Reply:
Oh sorry didn't realize you could see this. I'm just doing debug to try figure out an issue. Because right now, when I take out the line with pyinstaller, it causes issues. I narrowed it down to the package_windows.py with these print statements. I'm still in the process of debugging so I'll just make this a draft pr.

Reply:
To elaborate, the issue is with the second last line in the original file where it tries to set the current directory to src and it's unable to do so.

Reply:
Oh, okay. Sorry, I didn't realize you were still debugging. Okay, I'll take a look at the issue with the code.